src/sys/time.rs

```diff
@@ -143,6 +143,35 @@ impl TimeValLike for TimeSpec {
     }
 }
 
+#[cfg(any(
+    target_env = "uclibc",
+    target_env = "wasi",
+    target_os = "android",
+    target_os = "emscripten",
```

Comment:
You can add Dragonfly.  Besides that, what targets are missing from this list?  Does it not cover everything?

Reply:
Opened another PR: https://github.com/rust-lang/libc/pull/1487